pkg/server/plugin/datastore/sql/sql_test.go

```diff
@@ -787,6 +793,47 @@ func Test_GetPluginInfo(t *testing.T) {
 	require.NotNil(t, resp)
 }
 
+func Test_Migration(t *testing.T) {
+	require := require.New(t)
+
+	tmpDir, err := ioutil.TempDir("", "spire-sql-datastore")
+	require.NoError(err)
+	defer os.RemoveAll(tmpDir)
+
+	ds := New()
+
+	for i := 0; i < codeVersion; i++ {
+		dbName := fmt.Sprintf("v%d.sqlite3", i)
```

Comment:
dbname changes with every migration? Is this related to my above question?

Reply:
this code is copying a specifically named database out of the testdata folder on disk (i.e. v0.sqlite3, v1.sqlite3, etc.). The purpose is to add a database to the testdata folder for each database version so we can test migration from each version up to the latest.

Reply:
Ah yes, got it... sorry... carry on :)